### PR TITLE
Drop ruby-2.0 and mustermann19, use mustermann-grape instead

### PR DESCRIPTION
**Pull request by namusyaka**
ref #1469

### CODE REVIEW

**Comment by namusyaka** [comment]
OMG,  latest mustermann requires ruby-2.1 or later.
@dblock What do you think?

**Comment by dblock** [comment]
I say we go for it and drop 2.0. Rails already requires 2.2.

**Comment by dblock** [comment]
Does Rails 3 not work with Ruby 2.1? If so we should mention it and `git rm gemfiles/rails_3.gemfile` and remove it from Appraisals.

**Comment by dblock** [comment]
I release 0.17.0, maybe make a major version increment (replace 0.17.1 -> 0.18.0) with this PR?

**Comment by dblock** [comment]
Bump @namusyaka?

**Comment by namusyaka** [comment]
Sorry for the delay.  rails-3.2-stable seems like to support ruby 2.1.

>>I release 0.17.0, maybe make a major version increment (replace 0.17.1 -> 0.18.0) with this PR?

Yeah, the major version should be incremented, but I'd like to merge a few things for fixing bugs.

**Comment by namusyaka** [comment]
Just force pushed.

**Comment by dblock** [comment]
You also need to drop rails 3 from appraisals, no?

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,11 +13,13 @@
 #### Features
 
 * [#1393](https://github.com/ruby-grape/grape/pull/1393): Middleware can be inserted before or after default Grape middleware - [@ridiculous](https://github.com/ridiculous).
-* [#1390](https://github.com/ruby-grape/grape/pull/1390): Allow inserting middleware at arbitrary points in the middleware stack - [@rosa](https://github.com/rosa).
```
Comment (dblock) [inline CHANGELOG.md]: If you don't mind cleaning this up. Looks like some case-sensitive changes creeped in here that shouldn't be part of this commit.

Comment (namusyaka) [inline CHANGELOG.md]: OMG, sorry. fixed.

**Comment by namusyaka** [comment]
rails-3.2 and ruby-2.1 has passed specs on travis. https://travis-ci.org/ruby-grape/grape/jobs/158980775

**Comment by dblock** [comment]
Merged, thanks!
